### ISSUE TITLE
Tooltips appear inconsistently on Chrome and Safari

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#8040]
Can't reproduce this with Firefox, seems it happens only in Chrome and Safari. Haven't tested Edge and IE11 on Windows.

When hovering buttons, sometimes the tooltips don't appear as expected. For example, hovering the "Info" button in the top bar makes the tooltip "Content Structure:" appear:

<img width="377" alt="screen shot 2018-07-18 at 19 59 44" src="https://user-images.githubusercontent.com/1682452/42911695-55e2c69a-8aec-11e8-9a07-8736e08d2a96.png">

After that, depending on where the mouse pointer is moved, the tooltips of other buttons don't appear. To reproduce in Chrome or Safari:
- hover the "Info" button
- the tooltip appears
- move the mouse pointer horizontally to the "Plus" button on the left, making sure the mouse pointer stays within the area highlighted below while moving:

<img width="401" alt="screen shot 2018-07-18 at 19 59 53" src="https://user-images.githubusercontent.com/1682452/42911770-a89c51f8-8aec-11e8-888c-822acf166882.png">

- once on the "Plus" button, its tooltip doesn't appear (works with Firefox)
- repeat the steps above but this time move the mouse pointer outside the highlighted area before hovering the other button
- the tooltip appears correctly

When the Undo or Redo buttons are enabled, the behavior is slightly different but still not 100% correct (repeat the steps above multiple times and at some point, it will fail).

Wild guess: events propagating to some wrapper element?

**Comment by afercia** [comment]
Ok so this has to do with the issue mentioned in the inline comment in the tooltip component: https://github.com/facebook/react/issues/4251

Chrome and Safari don't fire the `mouseleave` event on disabled buttons. Not sure if React uses internally `mouseleave` or `mouseout` but it doesn't work regardless.  For the four buttons in the screenshot above, when Undo and Redo are disabled, tooltips can't work correctly.

However, this is also partially related to the implementation: there's a delay for the `isOver` state and these four buttons have no space between them. Spacing them helps (I've tried adding some margin for testing purposes) but it wouldn't be a proper solution. 

Some exploration for a different approach would be very welcome.

**Comment by karmatosed** [comment]
There are some inconsistency reports worth combining into this from here: https://github.com/WordPress/gutenberg/issues/11180

**Comment by afercia** [comment]
Needs to be retested: as far as I know, these buttons are not really disabled now.

**Comment by afercia** [comment]
Since these buttons no longer use `disabled` and use instead `aria-disabled`, the `mouseleave` event now works as expected. Tested in Chrome / Safari, the tooltips appear consistently.

An unintended, welcomed, consequence of #11379 and other changes to the other buttons. And one more reason to use `aria-disabled` + `noop` where it makes sense. /Cc @aduth
